theSrc/scripts/PlotlyChartElements.js

```diff
@@ -301,6 +305,19 @@ function addColorScale (trace, config) {
     trace['hoverlabel'].font = { color: hover_font_color }
 }
 
+function setMarkerColorsUsingScale (trace, config, panel_index) {
```

Comment:
Maybe we could rename this because it is not easy to see what its doing. How about `setTraceMarkerColorsFromConfig`?

Reply:
Done in [c1aced7](https://github.com/Displayr/rhtmlCombinedScatter/pull/67/commits/c1aced708983464d203f3a36caf27e61bd9c673c)